taxonomies/ingredients.txt

```diff
@@ -48486,6 +48553,10 @@ fr:assaisonnement saveur barbecue
 <en:condiment
 fr:assaisonnement goût salé
 
+<en:condiment
+es:salt and spices
+es:sal aromas
```

Comment:
"sal aromas" is probably not an ingredient, but an OCR issue (missing separator). So we should not add it.

Reply:
ok, removed